transformation_matrix:
  - -0.550799
  - -1.406152
  - 766.683889
  - -0.013610
  - -3.372605
  - 1659.777742
  - -0.000021
  - -0.002824
  - 1.000000
